node-cli/index.js

```diff
@@ -3,16 +3,25 @@ const inquirer = require("inquirer");
 const users = require("./users");
 const schemas = require("./schemas");
 const output = require("./output");
+const config = require("./config");
 
 /*  Change the logLevel to increase or decrease the 
     amount of messages you see in the console.
     Valid options are:
     fatal, error, warn, info, detail, debug, and trace
 */
-Realm.Sync.setLogLevel("error");
+const REALM_APP_ID = config.realmAppId;
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
+Realm.App.Sync.setLogLevel(app, "error");
 
 let realm;
 async function openRealm() {
+  try{
```

Comment:
Nitpick: Not sure if you have a standard eslint file, but I think we prefer a space after `try` before the `{`.